app/routes/customers_routes.py

```diff
@@ -0,0 +1,135 @@
+from flask import Blueprint
+from app import db
+from app.models.customer import Customer
+from app.models.rental import Rental
+from flask import Blueprint, jsonify, make_response, request, abort
+import datetime, requests, os
+import os
+
+customers_bp = Blueprint("customers_bp", __name__, url_prefix="/customers")
+
+# def valid_int(number, parameter_type):
```

Comment:
Remember to remove commented code.

Reply:
I imagine you did not use these helper methods because you needed the code to store running when you return a 400 level response. In this case, we can use `abort` in the helper functions.